hybrid/ddc_compartment_switching/shared.s

```diff
@@ -0,0 +1,138 @@
+// Copyright (c) 2021 The CapableVMs "CHERI Examples" Contributors.
+// SPDX-License-Identifier: MIT OR Apache-2.0
+
+.global compartment_simple_fun
+
+.text
+.balign 4
+.global switch_compartment
+.type switch_compartment, "function"
+switch_compartment:
+    // int switch_compartment(int a, int b, void * stack, size_t size)
```

Comment:
It'd be useful to update this comment, since it no longer reflects the C prototype.